packages/transport/src/core/TransportCoreErrors.ts

```diff
@@ -104,20 +104,13 @@ class Challenges {
 }
 
 class Datawallet {
-    public unsupportedModification(type: "unsupportedCacheChangedModificationCollection", data: any) {
-        const errorCode = "error.transport.datawallet.unsupportedModification";
+    public cacheChangeForUnsupportedCollectionDetected(data: any) {
```

Comment:
This renaming technically constitutes a breaking change and the code is included in the documentation, is it worth it?

Reply:
Docs yes, breaking no. I doubt that anyone relies on it.

Reply:
I think I have a solution to make it really backwards compatible.

Reply:
@Magnus-Kuhn changed, thanks for the heads-up!